### PR TITLE
Add tags to aws resources

### PR DESCRIPTION
**Pull request by danilosilvase**

### CODE REVIEW

**Review by fdammassa — CHANGES_REQUESTED** [review]
You missed the tags on `SLAViolationCloseSchedulingLambda` function; the commit seems not correctly aligned with develop. I suggest to create a new PR branching from develop again and reapplying the tags.

File: scripts/aws/cfn/storage.yml [hunk 1]
```diff
@@ -122,7 +126,6 @@ Resources:
       Parameters:
         QueueName: !Sub '${ProjectName}-progression-sensor-queue'
         DelaySeconds: 10
-        AlarmSNSTopicName: !Ref AlarmSNSTopicName
```
[the remark this answers is not in the document]
Comment (danilosilvase) [reply scripts/aws/cfn/storage.yml]: thanks for review it, my mistake

File: scripts/aws/cfn/storage.yml [hunk 2]
```diff
@@ -54,7 +54,7 @@ Resources:
           AttributeType: "S"
         - AttributeName: "type_endTimestampYearMonth"
           AttributeType: "S"
-        - AttributeName: "endTimeStamp"
+        - AttributeName: "endTimeStamp" # bool
```
[the remark this answers is not in the document]
Comment (danilosilvase) [reply scripts/aws/cfn/storage.yml]: updated

**Review by fdammassa — CHANGES_REQUESTED** [review]
The lambda `SLAViolationCloseSchedulingLambda` is still missing tags.

**Comment by danilosilvase** [comment]
> 
added tags, thanks

**Review by fdammassa — APPROVED** [review]

**Pull request closed without merging**